app/screens/contacts-screen/contacts.tsx

```diff
@@ -113,7 +113,7 @@ export const ContactsScreen: ScreenType = ({ navigation }: Props) => {
         }
       }
     `,
-    { skip: !hasToken },
+    { fetchPolicy: 'cache-and-network', skip: !hasToken },
```

Comment:
Does this work?  When I tried this policy on another query it didn't seem to change anything.

Reply:
It's meant to fix the case where the recipient of a payment from a new contact has `contacts` in their cache. When they mount this component, it will use the `contacts` in the cache while refetching, then use the updated `contacts` if they have changed.

So it's similar to `network-only` in what it solves for us here.